k8s-daemonset/k8s/hello-world.yml

```diff
@@ -50,7 +50,7 @@ metadata:
 spec:
   selector:
     app: hello
-  type: LoadBalancer
+  type: ClusterIP
```

Comment:
I think you can revert this change, since you're not using this config file for the ingress example.